packages/SwingSet/src/liveslots/liveslots.js

```diff
@@ -1315,11 +1364,14 @@ function build(
       // any promise it returns to fire.
       const p = Promise.resolve(delivery).then(unmeteredDispatch);
 
-      // Instead, we wait for userspace to become idle by draining the
-      // promise queue. We return 'p' so that any bugs in liveslots that
-      // cause an error during unmeteredDispatch will be reported to the
+      // Instead, we wait for userspace to become idle by draining the promise
+      // queue. We clean up and the return 'p' so that any bugs in liveslots
```

Comment:
typo "clean up and the return" should be "*then* return"

also please change the next line "bugs in liveslots that cause an error" to "bugs in liveslots *which* caues an error", for some reason the repeated "that" is bugging my grammar/ambiguity brain today

Reply:
Good catch on the typo.

On the matter of "that" vs. "which", in this case I think I can argue chapter and verse with whatever grammar gods you sacrifice to that the way it's written is correct, as the second "that" introduces a restrictive clause.  However, having two "that"s in close proximity definitely scans awkwardly.  I think the fix is actually to drop the first "that" entirely, i.e, "... then return 'p' so any bugs in liveslots that cause an error ..."

Also, I can't believe I just wrote an entire paragraph on a grammar issue.